PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD001.xml

```diff
@@ -0,0 +1,201 @@
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1014MD"/>
+                        <idno>MD001</idno>
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="126v">1-126</locus>
+                            <title type="complete" ref="LIT3178Deggwa"/> 
+                            <textLang xml:lang="gez"></textLang>
+                            <colophon xml:id="coloph1"><locus target="#1r"></locus>ናሁ፡ ወጠንኩ፡ ጽሒፈ፡ ጾመ፡ ድጓ፡ በሥምረተ፡ እግዚአብሔር፡
+                                በዘመነ፡ ቅዱስ፡ ዮሐንስ፡ ወንጌላዊ፡ <sic>በ፲፻፡ ወ፱፻፡ ወ ፵፡ </sic> ዓመተ፡ ምህረት፡ ለሥምረተ፡ እግዚአብሔር፡ አሜን፡ ወአሜን፡ ለይኩን፡ ለይኩን፡ 
+                                በመኃት፡ ወድራረ፡ ደም።                               
+                            </colophon>
+                            <colophon xml:id="coloph2"><locus target="#125v"></locus>ይትባረክ፡ እግዚአብሔር፡ <sic> ዘአዘአፈፀመኒ፡</sic> በደኅና፡ ወበሰላም፡
+                                በ፶፪፡ አ፡ ም፡ በዘመነ፡ ዮሐንስ፡                                  
+                            </colophon>
+                            <note>Used in the church school of <foreign xml:lang="gez">Zemā Bet</foreign></note>                                                           
+                              </msItem>
+                    </msContents>
+                    
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"></material>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">126</measure>
+                                    <measure unit="leaf" type="blank">3</measure>
+                                    <locus from="1" to="2"/> and <locus target="126"/>
+                                    <dimensions type="outer">
+                                        <height unit="mm">164</height>
+                                        <width unit="mm">132</width>
+                                        <depth unit="mm">51</depth>
+                                    </dimensions>
+                                </extent>     
+                                <foliation resp="AY">Foliation in pencil in the right of the bottom of the recto.</foliation>
+                                <collation>
+                                    <note>No quire marks</note>
```

Comment:
I think we do not note 'negative information', we just give quire numbers if there are any. but then I do not know whether our practices is the best :-)